src/vs/editor/contrib/codeAction/browser/codeActionModel.ts

```diff
@@ -197,12 +205,64 @@ export class CodeActionModel extends Disposable {
 					return;
 				}
 
-				const actions = createCancelablePromise(token => getCodeActions(this._registry, model, trigger.selection, trigger.trigger, Progress.None, token));
+				const startPosition = trigger.selection.getStartPosition();
+
+				const actions = createCancelablePromise(async token => {
+					if (this._showInlineQuickfxies()) {
+						const codeActionSet = await getCodeActions(this._registry, model, trigger.selection, trigger.trigger, Progress.None, token);
+
+						// Search for quickfixes in the curret code action set.
+						const foundQuickfix = codeActionSet.validActions?.some(action => action.action.kind === 'quickfix') || false;
```

Comment:
Use `CodeActionKind.QuickFix.contains(...)` here. That way we also match kinds like `quickfix.doStuff`